src/components/molecules/OakHandDrawnFocusUnderline/HandDrawnFocusUnderlineSvg.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
```

Comment:
The svg probably should be in an assets folder. It should be called InternalHandDrawnFocusUnderline

Reply:
or if it's not even exported from the folder ... SubHandDrawnFocusUnderline